jbmc/src/java_bytecode/java_entry_point.cpp

```diff
@@ -351,7 +351,7 @@ std::pair<code_blockt, std::vector<exprt>> java_build_arguments(
         init_code,
         symbol_table,
         factory_parameters,
-        lifetimet::AUTOMATIC_LOCAL,
```

Comment:
:bulb: I wonder if now all of the objects are dynamic if we can simplify the `allocate_objectst` to just be a function

Reply:
I don't know anything about it. If so, that could be a follow-up PR.